src/client/components/Title/TitleDesc.styles.css

```diff
@@ -0,0 +1,4 @@
+  .container{
```

Comment:
I think this class has been added alot of other places, and that can be very hard to debug. Can you name the class something more specific, e.g `title-desc-container`. 

Also, i'm not sure why we need absolute position?